### PR TITLE
Joomla! 3.5 and J3.x deprecation improvements.

### PR DESCRIPTION
**Pull request by xtech86**

### CODE REVIEW

File: component/admin/controllers/cpanel.php [hunk 1]
```diff
@@ -472,7 +472,9 @@ private function checkLanguagePackages()
 					{
 						$updateLanguagePackMessage = JText::sprintf('JEV_UPDATE_LANGUAGE_PACKAGE',$language['name']);
 					}
-					JError::raiseNotice("100", $updateLanguagePackMessage);
+
+					JFactory::getApplication()->enqueueMessage($updateLanguagePackMessage, 'warning');
```
Comment (GeraintEdwards) [inline component/admin/controllers/cpanel.php]: This should be a notice not a warning

Comment (xtech86) [inline component/admin/controllers/cpanel.php]: Yes! Sorry! updated now.

File: component/admin/jevents.php [hunk 2]
```diff
@@ -30,7 +31,7 @@
 $jver = explode( '.', $version->getShortVersion() );
 
 //version_compare(JVERSION,'1.5.0',">=")
-if (!isset($option))  $option = JRequest::getCmd("option"); // 1.6 mod
+if (!isset($option))  $option = $jinput->get("option"); // 1.6 mod
```
Comment (GeraintEdwards) [inline component/admin/jevents.php]: Should be getCmd not simple get

**Comment by GeraintEdwards** [comment]
Tony  - can you merge your last set of changes in so we can see what is left.

Thanks

**Pull request merged**